app/models/user.rb

```diff
@@ -146,6 +146,14 @@ def effective_responsible_bodies
     user_schools.map { |us| us.school&.responsible_body }.prepend(responsible_body).compact.uniq
   end
 
+  def computacenter_account_orgs
+    effective_responsible_bodies + further_education_schools
+  end
+
+  def further_education_schools
+    user_schools.map(&:school).filter { |s| s.is_a?(FurtherEducationSchool) }
```

Comment:
Can we do this with AR?
```suggestion
    schools.where(type: 'FurtherEducationSchool')
```

Reply:
Even better, this turned out to be dead code probably when we switched to having the placeholder RB so I will delete